services/twitter/twitter.service.js

```diff
@@ -118,7 +118,7 @@ class TwitterFollow extends BaseJsonService {
 
   async handle({ user }) {
     const data = await this.fetch({ user })
-    if (data.length === 0) {
+    if (data.length === undefined || data.length === 0) {
```

Comment:
Would this express the intent more clearly?

```suggestion
    if (! Array.isArray(data) || data.length === 0) {
```